File: BackEnd/postman/Enfermedad.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Enfermedad'
data={
  'accion':'crear',
  'idTipoEnfermedad': None,
  'enfermedad': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'agregarTipoEnfermedad',
  'descripcion': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'listarPorTipo',
  'idTipoEnfermedad': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'quitarDiagnostico',
  'idConsulta': None,
  'idExpediente': None,
  'idEnfermedad': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'actualizarTipoEnfermedad',
  'descripcion': None,
  'idTipoEnfermedad': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'listarTodos',
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'actualizar',
  'idTipoEnfermedad': None,
  'penfermedad': None,
  'idEnfermedad': None,
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'listarTipoEnfermedad',
}
print postear(data,url)
print

url='Enfermedad'
data={
  'accion':'diagnosticarEnfermedad',
  'idConsulta': None,
  'idEnfermedad': None,
}
print postear(data,url)
print

File: BackEnd/ajax/Enfermedad.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idTipoEnfermedad': null,
    'enfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTipoEnfermedad',
    'descripcion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorTipo',
    'idTipoEnfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'quitarDiagnostico',
    'idConsulta': null,
    'idExpediente': null,
    'idEnfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarTipoEnfermedad',
    'descripcion': null,
    'idTipoEnfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idTipoEnfermedad': null,
    'penfermedad': null,
    'idEnfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipoEnfermedad',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Enfermedad.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'diagnosticarEnfermedad',
    'idConsulta': null,
    'idExpediente': null,
    'idMedico': null,
    'idEnfermedad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


